### PR TITLE
event registration now saves changes

### LINKED ISSUE #1345: Can't set registration information fields from admin (for non-member registrations)
### Describe the bug
It is not possible to manually set the event registration information fields from the admin. This is quite problematic when adding non-member's registrations

### How to reproduce
Steps to reproduce the behaviour:
1. add a manual event registration
2. try to edit the information fields (this works)
3. you cannot save it

### Expected behaviour
It should save

### PR DESCRIPTION
**Pull request by mtsatsev**
Closes  #1345

### Summary
Organizers of events should be able to edit the members' fields at their events even for non-members. Now they can.

### How to test
Steps to test the changes you made:
1. Add a manual event registration.
1. Edit the information fields of the participants.
1. It works.

### CODE REVIEW

File: website/events/services.py [hunk 1]
```diff
@@ -187,8 +187,8 @@ def update_registration(
 
     if (
         not event_permissions(member, event, name)["update_registration"]
-        or not field_values
-    ):
+        and is_organiser(member, event)
```
Comment (se-bastiaan) [inline website/events/services.py:190]: When I am not an organiser this will always allow editing the event fields from the user perspective.

Comment (se-bastiaan) [reply website/events/services.py:190]: The change should probably in `event_permissions`. If you look at this PR: https://github.com/svthalia/concrexit/pull/1121/files, you see that the if statements have not the same meaning anymore. 

First, it checked for either an authenticated member or a name. Now it always requires a member first. That is more likely to be the problem here.

This change does not look like the right solution without causing new problems.

**Review by se-bastiaan — CHANGES_REQUESTED** [review]

File: website/events/services.py (hunk 1 above)
Comment (mtsatsev) [reply website/events/services.py:190]: Ok I will look it up again on Monday.

**Comment by se-bastiaan** [comment]
Closing this because of inactivity. Reopen when re-evaluated that we still want this change.

**Pull request closed without merging**